### PR TITLE
Starting a hotspot + example

### PR DESCRIPTION
**Pull request by majorz**
This is the part with creating a hotspot from #45

### CODE REVIEW

File: examples/hotspot.rs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+use network_manager::manager;
+use network_manager::device;
+use network_manager::connection;
+
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
+
+    if args.len() < 3 {
+        println!("USAGE: hotspot SSID PASSWORD");
```
Comment (josephroberts) [inline examples/hotspot.rs]: It would be nice if we could optionally pass in the wifi interface to be used, same with the password so the user can make an open hotspot

Comment (majorz) [inline examples/hotspot.rs]: Thanks, I will add those features now.

Comment (majorz) [inline examples/hotspot.rs]: Both features added in the latest commit

File: examples/hotspot.rs [hunk 2]
```diff
@@ -8,22 +8,78 @@ use network_manager::device;
...
     let args: Vec<String> = env::args().collect();
 
-    if args.len() < 3 {
-        println!("USAGE: hotspot SSID PASSWORD");
-        process::exit(1);
+    if args.len() < 2 {
+        print_usage_and_exit();
+    }
+
+    let (ssid_pos, interface) = if args[1] == "-i" {
```
Comment (josephroberts) [inline examples/hotspot.rs]: Do you think its worth replacing all of this with something like https://github.com/kbknapp/clap-rs ?

Or do you wanna do that in a seperate PR afterwards?

Comment (majorz) [inline examples/hotspot.rs]: Let's do that in a separate PR.
